content/en/docs/concepts/architecture/nodes.md

```diff
@@ -160,6 +160,15 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
+In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
+node. Starting from Kubernetes 1.13, node lease feature is introduced, so that
+both NodeStatus and node lease are treated as heartbeats. When node lease
+feature is enabled, node leases are renewed frequently while NodeStatus is
+reported from node to master only when there is some change or enough time has
+passed (longer than the default timeout of 40s). Since node lease is much more
```

Comment:
I think we set default to 1minute, right?

Reply:
Yes. I updated this sentence to make it more clear.